.codespellrc

```diff
@@ -1,3 +1,3 @@
 [codespell]
-skip = ./cover,./vendor,./omnibus,*.json,yarn.lock,seeds.exs,./**/node_modules,./deps,./priv/static,./priv/plts,./**/priv/static,./.git,./docs/build,./_build
+skip = ./erl_crash.dump,./apps/fz_http/erl_crash.dump,./cover,./vendor,./omnibus,*.json,yarn.lock,seeds.exs,./**/node_modules,./deps,./priv/static,./priv/plts,./**/priv/static,./.git,./docs/build,./_build
```

Comment:
I generally feel that we should add all developer-specific files to the [global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) file, there are as many different tools as developers and we don't want to overload the repo with configs for each of them.

Reply:
This is actually a linting configuration that runs for docs and code comments and has been pretty useful in those -- see https://github.com/firezone/firezone/blob/master/.github/workflows/static_analysis.yml#L90 and https://github.com/firezone/firezone/blob/master/.pre-commit-config.yaml#L27

Maybe we can try and keep these linter dotfiles more confined.

Reply:
I don't think all the tools we are using allow moving the config around, if so - we can later move them to `.dotfiles` and have all configs there, which will make PR look much cleaner.